### PR TITLE
Portfolio: setup and mobile version skeleton

### PR DESCRIPTION
**Pull request by Mehria-Rahmani**
-Modified README.md file.
-Added html markup for header and nav.
-Added styling for header.
-Added .ignore file for github actions, node_modules, linting files.
-Created package.json file.

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+.topnav {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  font-family: 'Roboto', sans-serif;
+  width: 100%;
+  position: fixed;
+  background: #3c3a39;
+}
```
Comment (omar-muhamad) [inline css/style.css]: - [ ] Please increase the height of the navbar and add more padding left and right to match the given Figma design.

| Figma Design  | Your Design |
| ------------- | ------------- |
| image  |image |

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+.job-title {
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 20px;
+  margin: 1px;
+  color: #fff5e1;
+}
```
Comment (omar-muhamad) [inline css/style.css:89]: - [ ] Please add the correct margin-top to the main header text to match the given Figma design.

| Figma Design  | Your Design |
| ------------- | ------------- |
| image | image  |

File: css/style.css [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+  gap: 4px;
+}
+
+.main-content {
+  padding: 50px 20px 1px 15px;
+}
+
+.headline {
+  font-style: normal;
+  font-family: 'Crete Round', sans-serif;
```
Comment (omar-muhamad) [inline css/style.css:71]: - [ ] I can see that you forgot to add the background image, So please add it as a `background-image` property to your headline section here in the CSS file you can know more about it [here](https://www.w3schools.com/cssref/pr_background-image.asp) and you can position it right following this [link](https://www.w3schools.com/cssref/pr_background-position.asp).

| Figma Design  | Your Design |
| ------------- | ------------- |
| image  | image |

**Review by omar-muhamad — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi @Mehria-Rahmani,

Nice job, you are almost there.



## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- Github flow is used correctly. ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- [ ] Please check the inline comments below.

## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by omar-muhamad** [comment]
Hi @Mehria-Rahmani,
After you ask for a review please don't push any new changes and wait for the review then kindly do the required changes asked by the reviewer.
So please consider all required changes I suggested above and push them then you can ask for another review with the same Pull request link.

Have a great day.

**Review by topeogunleye — CHANGES_REQUESTED** [review]
### Hi @Mehria-Rahmani, 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations
> 
> ## [Optional] suggestions:
> * Nothing to mention.
> 
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**